torch/csrc/stable/ops.h

```diff
@@ -201,4 +201,20 @@ inline Tensor zero_(Tensor& self) {
   return to<Tensor>(stack[0]);
 }
 
+// We expect this to be the stable version of the copy_ op with
+// identical semantics to the existing copy_ op (except that it will
+// not be called as a tensor method but only as a function
+// i.e. copy_(dst, src) not dst.zero_(src)).
```

Comment:
```suggestion
// identical semantics to the existing copy_ op
```
We can write the except note as a general statement somewhere, but let's not make this specific op different as the note applies generally.

Reply:
Suggestion applied.